Formula/forego.rb

```diff
@@ -16,15 +17,11 @@ class Forego < Formula
   end
 
   depends_on "go" => :build
-  depends_on "godep" => :build
 
   def install
     ENV["GOPATH"] = buildpath
-    mkdir_p buildpath/"src/github.com/ddollar/"
-    ln_sf buildpath, buildpath/"src/github.com/ddollar/forego"
-
-    ldflags = "-X main.Version=#{version} -X main.allowUpdate=false"
-    system "godep", "go", "build", "-ldflags", ldflags, "-o", bin/"forego"
+    ln_s "vendor", "src"
```

Comment:
Cool! Fixed in https://github.com/Homebrew/homebrew-core/pull/14154/commits/0b154fff0ffab8c7484c676a9722e2e39746d7cd